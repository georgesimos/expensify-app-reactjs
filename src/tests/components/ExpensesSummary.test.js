import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import selectExpenses from '../../selectors/expenses-total';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should ExpensesSummary render correctly with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount = {1} expenseTotal = {400}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should ExpensesSummary render correctly with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary  expenseCount = {23} expenseTotal = {40067}/>);
    expect(wrapper).toMatchSnapshot();
});
