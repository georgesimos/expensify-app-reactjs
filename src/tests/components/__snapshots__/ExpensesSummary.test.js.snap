// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should ExpensesSummary render correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $400
     
  </h1>
</div>
`;

exports[`should ExpensesSummary render correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $40,067
     
  </h1>
</div>
`;
